tests/foreman/api/test_user.py

```diff
@@ -55,28 +62,7 @@ def test_create(self, attrs):
             self.assertIn(name, real_attrs.keys())
             self.assertEqual(value, real_attrs[name])
 
-    @decorators.data(
-        {u'admin': False},
-        {u'admin': True},
-        {u'firstname': gen_string('alphanumeric', randint(1, 50))},
-        {u'firstname': gen_string('alpha', randint(1, 50))},
-        {u'firstname': gen_string('cjk', randint(1, 50))},
-        {u'firstname': gen_string('latin1', randint(1, 50))},
-        {u'firstname': gen_string('numeric', randint(1, 50))},
-        {u'firstname': gen_string('utf8', randint(1, 16))},
-        {u'lastname': gen_string('alphanumeric', randint(1, 50))},
-        {u'lastname': gen_string('alpha', randint(1, 50))},
-        {u'lastname': gen_string('cjk', randint(1, 50))},
-        {u'lastname': gen_string('latin1', randint(1, 50))},
-        {u'lastname': gen_string('numeric', randint(1, 50))},
-        {u'lastname': gen_string('utf8', randint(1, 16))},
-        {u'login': gen_string('alphanumeric', randint(1, 100))},
-        {u'login': gen_string('alpha', randint(1, 100))},
-        {u'login': gen_string('cjk', randint(1, 100))},
-        {u'login': gen_string('latin1', randint(1, 100))},
-        {u'login': gen_string('numeric', randint(1, 100))},
-        {u'login': gen_string('utf8', randint(1, 33))},
-    )
+    @decorators.data(*_user_attrs())
```

Comment:
Why will `_user_attrs()` not provide the same data set? The first commit message shows that a full 40 tests are still run after all this data is pushed into the `_user_attrs()` function.